lib/src/widgets/iframe_mobile.dart

```diff
@@ -27,35 +28,100 @@ class IframeContentElement extends ReplacedElement {
 
   @override
   Widget toWidget(RenderContext context) {
-    final sandboxMode = attributes["sandbox"];
-    return Container(
+    return SizedBox(
       width: width ?? (height ?? 150) * 2,
       height: height ?? (width ?? 300) / 2,
       child: ContainerSpan(
         style: context.style,
         newContext: context,
-        child: webview.WebView(
-          initialUrl: src,
-          key: key,
-          javascriptMode: sandboxMode == null || sandboxMode == "allow-scripts"
-            ? webview.JavascriptMode.unrestricted
-            : webview.JavascriptMode.disabled,
-        navigationDelegate: (request) async {
-          final result = await navigationDelegate!(NavigationRequest(
-            url: request.url,
-            isForMainFrame: request.isForMainFrame,
-          ));
-          if (result == NavigationDecision.prevent) {
-            return webview.NavigationDecision.prevent;
-          } else {
-            return webview.NavigationDecision.navigate;
-          }
-        },
-          gestureRecognizers: {
-            Factory<VerticalDragGestureRecognizer>(() => VerticalDragGestureRecognizer())
-          },
+        child: _WebViewWidget(
+          src: src,
+          navigationDelegate: navigationDelegate,
+          attributes: attributes,
         ),
       ),
     );
   }
-}
\ No newline at end of file
+}
+
+class _WebViewWidget extends StatefulWidget {
+  const _WebViewWidget({
+    Key? key,
+    this.src,
+    required this.attributes,
+    required this.navigationDelegate,
+  }) : super(key: key);
+
+  final String? src;
+  final Map<String, String> attributes;
+  final NavigationDelegate? navigationDelegate;
+
+  @override
+  State<_WebViewWidget> createState() => _WebViewWidgetState();
+}
+
+class _WebViewWidgetState extends State<_WebViewWidget> {
+  late final webview.WebViewController controller;
+  late final webview.PlatformWebViewControllerCreationParams params;
+  final UniqueKey key = UniqueKey();
+
+  @override
+  void initState() {
+    super.initState();
+    if (webview.WebViewPlatform.instance is WebKitWebViewPlatform) {
+      params = WebKitWebViewControllerCreationParams(
+        allowsInlineMediaPlayback: true,
+        mediaTypesRequiringUserAction: const <PlaybackMediaTypes>{},
+      );
```

Comment:
This setting does not seem to exist before the migration. Could you let me know why it is needed? 🙇

Reply:
@m-sugawara Thanks for the review 🙏 
- this is not needed 🙇 
- i have updated the PR to remove the unnecessary changes.